presto-main/src/main/java/com/facebook/presto/operator/aggregation/AccumulatorCompiler.java

```diff
@@ -554,12 +555,12 @@ private static void generateAddIntermediateAsCombine(
         Variable thisVariable = method.getThis();
 
         Variable block = scope.getVariable("block");
-        Variable scratchState = scope.declareVariable(singleStateClass, "scratchState");
+        Variable scratchState = scope.declareVariable(stateClass, "scratchState");
         Variable position = scope.declareVariable(int.class, "position");
-
-        body.comment("scratchState = stateFactory.createSingleState();")
+        String createStateMethodName = grouped ? "createGroupedState" : "createSingleState";
+        body.comment("scratchState = stateFactory.createSingleState/createGroupeState();")
```

Comment:
Hey, I've just noticed a typo here: `s/createGroupeState/createGroupState/`

Reply:
nice catch--i wonder how it worked...
pretty sure grouped == true happened,it's how i came to find this issue

thanks

Reply:
nvm, it was in the comment of generated code
fixed anyway, thanks